### PR TITLE
Use new DataTypes from portability-client-types in SetupHandler. 

### PR DESCRIPTION
**Pull request by seehamrun**
For using the new data types module (#71)
This requires a new binary and static content build (#67)

### CODE REVIEW

File: portability-api/src/main/java/org/dataportabilityproject/webapp/SetupHandler.java [hunk 1]
```diff
@@ -180,9 +183,7 @@ JsonObject handleCopySetup(Headers requestHeaders, PortabilityJob job) {
 
     }
 
-    return Json.createObjectBuilder().add(JsonKeys.DATA_TYPE, job.dataType())
-        .add(JsonKeys.EXPORT_SERVICE, job.exportService())
-        .add(JsonKeys.IMPORT_SERVICE, job.importService()).build();
+    return new DataTransferResponse(job.exportService(), job.importService(), job.dataType(), Status.INPROCESS, "/startCopy");
```
Comment (holachuy) [inline portability-api/src/main/java/org/dataportabilityproject/webapp/SetupHandler.java]: Either now or as a TODO, let's make /startCopy a constant in a shared class so we can keep all our urls in a central place

File: portability-api/src/main/java/org/dataportabilityproject/webapp/SetupHandler.java [hunk 2]
```diff
@@ -56,6 +59,7 @@
   private final Mode mode;
   private final String handlerUrlPath;
   private final TokenManager tokenManager;
+  private final ObjectMapper objectMapper;
```
Comment (holachuy) [inline portability-api/src/main/java/org/dataportabilityproject/webapp/SetupHandler.java]: Can this be static final?

Comment (seehamrun) [reply portability-api/src/main/java/org/dataportabilityproject/webapp/SetupHandler.java]: Done (also made logger final static)

File: portability-api/src/main/java/org/dataportabilityproject/webapp/SetupHandler.java (hunk 1 above)
Comment (seehamrun) [reply portability-api/src/main/java/org/dataportabilityproject/webapp/SetupHandler.java]: Ack - I'll do that in my next PR

**Pull request merged**